ansible/requirements.yml

```diff
@@ -126,7 +126,7 @@
   version: 0.1.0
 
 - src: git+https://github.com/openmicroscopy/ansible-role-devspace.git
-  version: master
+  version: 0.2.1
```

Comment:
Should we pull from Galaxy then i.e. `src: openmicroscopy.devspace` ?

Reply:
I was thinking about that.
If we are all okay with that it will make more sense